### PR TITLE
Multiple small fixes

### PR DESCRIPTION
**Pull request by ArjenMiedema**
I've removed the modman file, because there were some issues when using composer. This version has composer mapping, so now it's fully composer based.

Also, the composer.json contains more info now and all files are in a src subfolder, to keep them separated from the non-extension files.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,9 +1,26 @@
 {
+    "name": "mediact/footerjs",
```
Comment (bobbyshaw) [inline composer.json]: Why the rename?

File: modman [hunk 2]
```diff
@@ -1,2 +0,0 @@
-app/code/community/Meanbee/Footerjs/ app/code/community/Meanbee/Footerjs/
```
Comment (bobbyshaw) [inline modman]: What problem did you run into using modman in collaboration with composer? I'm not comfortable moving from support one installation method to another. Composer isn't standard for many on M1.

File: composer.json (hunk 1 above)
Comment (ArjenMiedema) [reply composer.json]: This was unintentional. Will fix it.

File: modman (hunk 2 above)
Comment (ArjenMiedema) [reply modman]: I agree. Got the feedback from colleagues in the past weeks, mentioning that without modman there was no issue. After reusing it myself today, it looks like it was related to an issue mentioned earlier today by Andrew Howden on StackExchange for Magento 2 (also applies here): https://magento.stackexchange.com/questions/177924/magenta-2-1-5-composer-install-fails-to-generate-all-files. I'll put it back where it belongs.

File: modman [hunk 3]
```diff
@@ -0,0 +1,2 @@
+src/app/code/community/Meanbee/Footerjs src/app/etc/modules/Meanbee_Footerjs.xml
```
Comment (bobbyshaw) [inline modman]: This doesn't look quite right.

**Comment by bobbyshaw** [comment]
Thanks @ArjenMiedema!

**Pull request merged**